graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/requests/IndexSetUpdateRequest.java

```diff
@@ -75,6 +75,21 @@
     @NotNull
     public abstract RetentionStrategyConfig retentionStrategy();
 
+    @JsonProperty("index_analyzer")
+    @NotBlank
+    public abstract String indexAnalyzer();
+
+    @JsonProperty("index_template_name")
```

Comment:
I think we shouldn't allow the update of the index template name for an index set. This makes it harder to cleanup the template when an index set gets deleted.